x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/search_index.tsx

```diff
@@ -144,11 +149,17 @@ export const SearchIndex: React.FC = () => {
       isLoading={indexApiStatus === Status.LOADING || indexApiStatus === Status.IDLE}
       pageHeader={{
         pageTitle: indexName,
-        rightSideItems: indexData?.crawler ? [<CrawlerStatusIndicator />] : [],
+        rightSideItems: [
+          ...headerActions,
+          ...(indexData?.crawler ? [<CrawlerStatusIndicator />] : []),
```

Comment:
I wonder if this logic to include the status indicator should move into the new `header_actions.tsx` file.